README.md

```diff
@@ -7,6 +7,8 @@ They retrieve data from the following sites:
 - Foreign doctors: http://medicalboard.co.ke/online-services/foreign-doctors-license-register
 - Clinical officers: http://clinicalofficerscouncil.org/online-services/retention/
 - Health Facilities: http://kmhfl.health.go.ke
+- NHIF accredited facilities: http://www.nhif.or.ke/healthinsurance/medicalFacilities
+- NHIF accredited inpatient facilities: http://www.nhif.or.ke/healthinsurance/inpatientServices
```

Comment:
You can fin outpatient here - http://www.nhif.or.ke/healthinsurance/outpatientServices